README.md

```diff
@@ -19,6 +19,22 @@ A comprehensive proofs handler for Ethereum trie. Tested with various EIPs inclu
   - [x] Retrieve proof by transaction index
   - [x] Verify proof
 
+## CLI Tool
+
+The CLI tool supports generating proofs for transactions and receipts. Use the following commands based on your requirements:
+
+**Generate a Proof via CLI**
+To generate a proof for a transaction, use the following command:
+
+`cargo run --bin eth-trie-proofs tx <TRANSACTION_HASH> [RPC_URL]`
```

Comment:
better to install cli binary and then run 
e.g base on current repo structure
```
cargo install --path ./
```
and then run binary 
```
eth-trie-proofs --help
```

Reply:
further suggestion: 
restructure cli binary into seperate crate
```
bin/{binary_name}/main.rs 
bin/{binary_name}/Cargo.toml
```

small suggestion: 
`eth-trie-proofs` seems bit long for binary name, mb we can shorten into smth else `etp` `ettp` ?